web-ui/src/context/selectors.js

```diff
@@ -36,103 +47,58 @@ export const selectIsAdmin = createSelector(
-    userProfile.permissions.some(p => p?.permission?.includes('REPORT'))
-);
+export const selectHasReportPermission = hasPermission('REPORT');
 
-export const selectHasAnniversaryReportPermission = createSelector(
-  selectUserProfile,
-  userProfile =>
-    userProfile &&
-    userProfile.role &&
-    userProfile.permissions.some(p =>
-      p?.permission?.includes('CAN_VIEW_ANNIVERSARY_REPORT')
-    )
-);
+export const selectHasAnniversaryReportPermission =
+  hasPermission('CAN_VIEW_ANNIVERSARY_REPORT');
 
-export const selectHasBirthdayReportPermission = createSelector(
-  selectUserProfile,
-  userProfile =>
-    userProfile &&
-    userProfile.role &&
-    userProfile.permissions.some(p =>
-      p?.permission?.includes('CAN_VIEW_BIRTHDAY_REPORT')
-    )
-);
+export const selectHasBirthdayReportPermission =
+  hasPermission('CAN_VIEW_BIRTHDAY_REPORT');
 
-export const selectHasCheckinsReportPermission = createSelector(
-  selectUserProfile,
-  userProfile =>
-    userProfile &&
-    userProfile.role &&
-    userProfile.permissions.some(p =>
-      p?.permission?.includes('CAN_VIEW_CHECKINS_REPORT')
-    )
-);
+export const selectHasCheckinsReportPermission =
+  hasPermission('CAN_VIEW_CHECKINS_REPORT');
 
-export const selectHasProfileReportPermission = createSelector(
-  selectUserProfile,
-  userProfile =>
-    userProfile &&
-    userProfile.role &&
-    userProfile.permissions.some(p =>
-      p?.permission?.includes('CAN_VIEW_PROFILE_REPORT')
-    )
-);
+export const selectHasProfileReportPermission =
+  hasPermission('CAN_VIEW_PROFILE_REPORT');
 
-export const selectHasPulseReportPermission = createSelector(
-  selectUserProfile,
-  userProfile =>
-    userProfile &&
-    userProfile.role &&
-    userProfile.permissions.some(p =>
-      p?.permission?.includes('CAN_VIEW_ALL_PULSE_RESPONSES')
-    )
-);
+export const selectHasViewPulseReportPermission =
+  hasPermission('CAN_VIEW_ALL_PULSE_RESPONSES');
 
-export const selectHasSkillsReportPermission = createSelector(
-  selectUserProfile,
-  userProfile =>
-    userProfile &&
-    userProfile.role &&
-    userProfile.permissions.some(p =>
-      p?.permission?.includes('CAN_VIEW_SKILLS_REPORT')
-    )
-);
+export const selectHasSkillsReportPermission =
+  hasPermission('CAN_VIEW_SKILLS_REPORT');
 
-export const selectHasTeamSkillsReportPermission = createSelector(
-  selectUserProfile,
-  userProfile =>
-    userProfile &&
-    userProfile.role &&
-    userProfile.permissions.some(p =>
-      p?.permission?.includes('CAN_VIEW_SKILLS_REPORT')
-    )
-);
+export const selectHasTeamSkillsReportPermission =
+  hasPermission('CAN_VIEW_SKILLS_REPORT');
 
-export const selectHasUpdateReviewAssignmentsPermission = createSelector(
-  selectUserProfile,
-  userProfile =>
-    userProfile &&
-    userProfile.role &&
-    userProfile.permissions.some(p =>
-      p?.permission?.includes('CAN_UPDATE_REVIEW_ASSIGNMENTS')
-    )
-);
+export const selectHasCreateReviewAssignmentsPermission =
```

Comment:
I added selectors for all the new permissions related to review assignments and review periods.
Maybe Tim is removing some of these.

Reply:
I think all these are staying 👍